components/Buttons/src/MDCButton.m

```diff
@@ -515,6 +558,18 @@ - (void)setInkMaxRippleRadius:(CGFloat)inkMaxRippleRadius {
   _inkView.maxRippleRadius = inkMaxRippleRadius;
 }
 
+- (void)setEnableRippleBehavior:(BOOL)enableRippleBehavior {
+  _enableRippleBehavior = enableRippleBehavior;
+
+  if (enableRippleBehavior) {
+    [self.inkView removeFromSuperview];
+    [self insertSubview:self.rippleView belowSubview:self.imageView];
+  } else {
+    [self.rippleView removeFromSuperview];
+    [self insertSubview:self.inkView belowSubview:self.rippleView];
```

Comment:
was this supposed to be `belowSubview:self.imageView`?

Reply:
Whoops! Good catch, thanks